front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -1284,7 +1284,7 @@ function SlackIntegration({
               }}
               expandable={false}
               fullySelected={false}
-              filterPermission="write"
+              filterPermission="none"
```

Comment:
Note: I don't think we have used `none` (it is omitted in the Add/Remove data screen of connections) but this should work similarly,